pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.uid2</groupId>
     <artifactId>uid2-admin</artifactId>
-    <version>2.10.0-7a209d003f</version>
+    <version>2.10.4-a38de4d955</version>
```

Comment:
Why manually update the version here? I thought our GH action will do that for us

Reply:
That's from me doing a branch build before